dialogue-core/src/main/java/com/palantir/dialogue/core/BalancedScoreTracker.java

```diff
@@ -62,7 +61,7 @@
 
     BalancedScoreTracker(
             int channelCount, Random random, Ticker ticker, TaggedMetricRegistry taggedMetrics, String channelName) {
-        Preconditions.checkState(channelCount >= 2, "At least two channels required");
+        //        Preconditions.checkState(channelCount >= 2, "At least two channels required");
```

Comment:
@carterkozak thoughts on this? I'd like to simulate it based on the BalancedScoreTracker, but need to make sure that both request flows land on the same host. So I just made the simulation have a single host, but then this piece complains...

Reply:
I don't think that check was ever really necessary, nothing wrong with using the mechanism for a single channel even if it doesn't always make sense.